mtp_noms_ops/apps/security/tests/test_forms/test_search_forms.py

```diff
@@ -1329,7 +1329,7 @@ def test_invalid(self):
                     'payment_method': PaymentMethod.online.name,
                     'card_number_last_digits': '12345',
                 },
-                {'card_number_last_digits': ['You’ve entered too many characters']},
+                {'card_number_last_digits': ['Ensure this value has at most 4 characters (it has 5).']},
```

Comment:
the previous message used to come from mtp-common so i don't think this is right

Reply:
https://github.com/ministryofjustice/money-to-prisoners-common/blob/main/mtp_common/forms/__init__.py#L11-L20

Reply:
ah i think it's cos you didn't move the app config https://github.com/ministryofjustice/money-to-prisoners-common/blob/main/mtp_common/app.py#L7

Reply:
Right! Well spotted, I missed that one - I didn't realise we override some of the forms error messages.

[I've made this change in this (draft) PR](https://github.com/ministryofjustice/money-to-prisoners-common/pull/693): I'm retesting the apps locally and reverting the commits to change the form messages tests in the various apps.